src/OAuth2User.ts

```diff
@@ -3,7 +3,7 @@ import { buildQueryString, basicAuthHeader } from "./utils";
 import { OAuthClient, AuthHeader, GetTokenResponse, Token, GenerateAuthUrlOptions } from "./types";
 import { RequestOptions, rest } from "./request";
 
-const AUTHORIZE_URL = "https://getalby.com/oauth";
+let AUTHORIZE_URL = "https://getalby.com/oauth";
```

Comment:
could you please keep this as a constant?

Reply:
resolved